html/rendering/non-replaced-elements/the-fieldset-element-0/legend-dynamic-update.html

```diff
@@ -1,10 +1,19 @@
 <!DOCTYPE html>
+<html class=reftest-wait>
 <title>legend and dynamic update</title>
 <link rel=fieldset-foo-ref.html>
 <p>There should be a normal fieldset below with the legend "Foo".</p>
 <fieldset>
   <legend>F</legend>
 </fieldset>
 <script>
-  document.querySelector('legend').textContent += "oo";
+  const legend = document.querySelector('legend');
+  // force layout
+  legend.offsetTop;
+  requestAnimationFrame(() => {
```

Comment:
The requestAnimationFrame() stuff isn't really needed for this test, but if you insist... :)

Reply:
Can't hurt :-)